AndelaOSP/wire-api
# #167599208 Update lodash and axios dependencies

### What does this PR do?
- This PR updates the lodash and axios dependencies

#### Description of Task to be completed?
- Run `yarn audit` to update the dependencies
- Manually install current versions of the dependencies

#### How should this be manually tested?
- Clone this repository.
- Go to the terminal and checkout to the branch `ch-update-dependencies-167599208`
- Change directories to the `src` folder
- Install all dependencies using `yarn install`
- Run the yarn audit command and confirm that there are no vulnerabilities of the `lodash` and `axios` dependencies.
- Run the tests using `yarn test`
- All the tests should run successfully

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#167599208](https://www.pivotaltracker.com/n/projects/2117172/stories/167599208)

#### Screenshots (if appropriate)
N/A
